pending.py

```diff
@@ -68,13 +68,17 @@ def iterate_prs(repo, skip, users, path):
             print_color('red', "SKIP untrusted user")
         elif (user, branch) in skip:
             print_color('yellow', "SKIP")
-        elif pr.base.ref != args.main:
-            print_color('yellow', "SKIP base: {}".format(pr.base.ref))
+        elif (pr.base.ref != args.main and
+              (pr.base.ref not in include_spec['search_branches'] and
+               not set(labels) ^ set(include_spec['include_labels']))):
+            print_color('yellow', "SKIP base: {} labels: {}".format(pr.base.ref, labels))
         elif args.skip_label and args.skip_label in labels:
             print_color('yellow', "SKIP label: {}".format(args.skip_label))
         else:
             if pull(args, pr, path):
-                print_color('green', "OK")
+                print_color('green',
+                            "OK base: {} labels: {}".format(pr.base.ref,
+                                                            labels))
```

Comment:
Based on my previous comment, I would suggest to leave this unchanged.

Reply:
OK, I seems reasonable to keep things simple. May be got biased as I used it for testing purposes, so  distinguishing between commits from `args.main` and other branches was something I found useful.